### ISSUE TITLE
[Select]: A11y No Focus within list item with TalkBack enabled

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11485]
### Bug Description

With Talkback enabled when user expands list box ‘Select’ screen is open but there is no focus available within the list item and with swipe gesture it is observed that the focus remains on the parent screen UI elements.

![Image](https://github.com/user-attachments/assets/99e0c4ee-5566-4410-ac10-e37aa74b25c4)

### Affected Component

Select

### Expected Behaviour

When user expands the list box the list items must receive focus.

### Steps to Reproduce

1. Open example on mobile device https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/inputs-select--default
2. Enable TalkBack on your mobile device.
3. Use swipe gestures to navigate to Select component.
4. Double click to open list items.
5. Observe that the focus remains on the parent screen.

### Priority

None

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Operating System

Android

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by AlekseiSch** [comment]
@Valkanister also reproducible with VoiceOver. Focus is displayed behind the dialog

![Image](https://github.com/user-attachments/assets/93a69008-7d2f-487d-9897-a7ab1e9be0b6)

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-rd, 

I think this is a known problem with dialogs/popovers on mobile screens where the focus is left behind. Please have a look. 
Kind regards, 

Georgi

SAP Development Support

**Comment by ArtemJJS** [comment]
HI @Valkanister , any news on that topic?

**Comment by GerganaKremenska** [comment]
Hello @ArtemJJS,

We cannot control where the focus goes in this particular case. UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of device, browser and screen reader.

Best,
Gergana
